components/FileUpload.jsx

```diff
@@ -1,7 +1,7 @@
 import React, { useCallback } from 'react';
 import { useDispatch } from 'react-redux';
 import { useDropzone } from 'react-dropzone';
-import { parse } from '@loaders.gl/core';
+import { parse } from '@loaders.gl/core'; // TODO: use parseinbatches?
 import { CSVLoader } from '@loaders.gl/csv';
```

Comment:
👍  Please also test @loaders.gl/gis for geojson and @loaders.gl/shapefile for shapefile
https://loaders.gl/docs/specifications/category-gis
https://loaders.gl/docs/modules/shapefile/api-reference/shapefile-loader